tests/SingleTableInheritanceTraitModelMethodsTest.php

```diff
@@ -79,13 +79,13 @@ public function testGetQualifiedSingleTableTypeColumn() {
   // getSingleTableTypes
 
   public function testGetSingleTableTypesOfRoot() {
-    $types = ['motorvehicle', 'car', 'truck', 'bike'];
+    $types = ['motorvehicle', 'bike', 'car', 'taxi', 'truck'];
 
```

Comment:
To me it's correct as a taxi is a type of Vehicle.  The only other change is the ordering but that's only down to how the type map is built.